types/account.d.ts

```diff
@@ -158,10 +168,6 @@ export type UpdateCustomerConsentsPayload = {
   consents: CustomerConsent[];
 };
 
-export type RefreshTokenPayload = {
```

Comment:
Why is this deleted?

Reply:
Ugh, I guess somehow when was trying to move the types..